README.md

```diff
@@ -2,7 +2,8 @@
 
 [![NCSA licensed](https://img.shields.io/badge/license-NCSA-blue.svg)](LICENSE)
 [![CI](https://github.com/ILLIXR/ILLIXR/workflows/illixr-tests-master/badge.svg)](https://github.com/ILLIXR/ILLIXR/actions)
```

Comment:
Need someone who can login into Jenkins to update the status of CI (ref: https://shields.io/badges/jenkins). I.e. I would need the Job URL (Example: https://wso2.org/jenkins/view/All%20Builds/job/archetypes)

Reply:
I could probably do that, but I do not have access to the example web page, so I do not know what particulars are needed.

Reply:
I could not log into Jenkins as well. @shg8 can you help me get the link to the Jenkins build?